api/src/routes/anime/index.ts

```diff
@@ -0,0 +1,9 @@
+import express from 'express';
+import animeRoutes from '@@root/routes/anime/animeRoutes';
```

Comment:
This is a nit pick, so you don't have to actually change anything here, but you don't have to use `@@root`, you should just be able to use `@@routes/anime/animeRoutes.js`

Reply:
If you don't change it, you should still add the `.js` to the end of `animeRoutes`